kclvm/tools/src/LSP/src/find_refs.rs

```diff
@@ -13,46 +14,41 @@ use std::collections::HashMap;
 use std::sync::Arc;
 
 pub(crate) fn find_refs<F: Fn(String) -> Result<(), anyhow::Error>>(
-    program: &Program,
```

Comment:
Unit test failed in CI: https://github.com/kcl-lang/kcl/actions/runs/6939682497/job/18877398025?pr=899